### DISCUSSION TITLE
Integrate Sentry Vite Plug-in

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#664] [Announcements]
Sentry has a vite plugin which allows us to delete some of our custom integration code and upload build source maps during the docker build rather than before deploys which is an improvement!

Check out the pull request here: https://github.com/epicweb-dev/epic-stack/pull/660

Big thanks to @devlinmorrow, @johnpaulkiser, and @fredericrous!

**Comment by hilja** [comment]
I was tinkering with this in my Remix project but opted out because it's yet another dependency, and sentry cli comes with the `@sentry/remix` package. Running it in the `release_command` is pretty straight forward:

```shell
# Upload source maps to Sentry and make a release
pnpm sentry-upload-sourcemaps \
  --org='foo' \
  --project='foo' \
  --buildPath='build' \
  --urlPrefix='#build/' \
  --release="$(echo $APP_REVISION)"
```

It doesn't always remove the maps though, so better following it with a `find`:

```shell
find ./build -type f -name '*.map' -delete
```
